tests/helpers/submissionListHelper-test.js

```diff
@@ -0,0 +1,7 @@
+// No tests needed ATM; all API Requests
+// parseRecentActivity?
```

Comment:
There was a bug that came up recently related to file size calculations (some of which happen in `parseRecentActivity`), so it should have unit tests. Or we move a generic file size calculation to it's own helper function.